robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -75,6 +123,11 @@ public double getPipelineLatency() {
         return corners;
     }
 
+    /*
+     * Set the operating mode of the Limelight, either VISION (which runs the pipeline) or DRIVER (which disables
```

Comment:
Instead of specifying what each enum value does in the method docs, add docs to the enum values

Reply:
dnoe